content/en/blog/2019/announcing-1.3/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Announcing Istio 1.3
+subtitle: Major Update
+description: Istio 1.3 release announcement.
+publishdate: 2019-09-10
+attribution: The Istio Team
+release: 1.3.0
+---
+
+We are pleased to announce the release of Istio 1.3!  
+
+{{< relnote linktonote="true" >}}
+
+The theme of Istio 1.3 is User Experience:
+
+- Improve the experience of new users adopting Istio
+- Improve the experience of users debugging problems
+- Support more applications without any additional configuration
+
+Every few releases, the Istio team gets excited as we make dramatic improvements to usability, APIs, and the overall system performance. This release is one of those where the team is excited to roll out some key updates:
+
+*Intelligent protocol detection*: In past releases, you had to explicitly declare the protocol for service ports by using a special port naming format. This requirement caused problems for users that did not name their ports when they added their applications to the mesh. Starting with 1.3, the protocol on the wire will be automatically detected (as HTTP or TCP), eliminating the need to name ports explicitly. We will be adding support for additional [non-HTTP] protocols in future releases. 
```

Comment:
Replace all the paragraph lead-ins with real sections:

## Intellifent protos detection

Reply:
This is not entirely accurate as I just hit this today.  User will still need to name the ports when there are more than 1 port.  It is just we don't enforce more requirements than what K8s has and there is no need to prefix the name with protocol.   Also, this is only for http and tcp for 1.3.  So it should be something like "eliminating the need to prefix the port name explicitly for HTTP or TCP."

Reply:
Done